packages/sku/tsconfig.json

```diff
@@ -14,5 +14,11 @@
     "declaration": true
   },
   "include": ["./src/**/*"],
-  "exclude": ["node_modules", "**/@loadable"]
+  "exclude": [
+    "node_modules",
+    "**/@loadable",
+    "**/*.test.ts",
+    "**/*.test.tsx",
+    "**/*.test.js"
+  ]
```

Comment:
This was undone in the previous PR. Was it intentionally changed back in this PR?